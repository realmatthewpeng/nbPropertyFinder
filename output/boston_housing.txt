Total number of lines found: 0


Incorrect Lines:


Precision = 0
Recall = 0.0

features = data.drop('MEDV', axis = 1)
print "Boston housing dataset has {} data points with {} variables each.".format(*data.shape)
print "Minimum price: ${:,.2f}".format(minimum_price)
print "Maximum price: ${:,.2f}".format(maximum_price)
print "Mean price: ${:,.2f}".format(mean_price)
print "Median price ${:,.2f}".format(median_price)
print "Standard deviation of prices: ${:,.2f}".format(std_price)
