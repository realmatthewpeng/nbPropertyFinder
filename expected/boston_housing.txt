features = data.drop('MEDV', axis = 1)
print "Boston housing dataset has {} data points with {} variables each.".format(*data.shape)
print "Minimum price: ${:,.2f}".format(minimum_price)
print "Maximum price: ${:,.2f}".format(maximum_price)
print "Mean price: ${:,.2f}".format(mean_price)
print "Median price ${:,.2f}".format(median_price)
print "Standard deviation of prices: ${:,.2f}".format(std_price)